apache/spark
# [SPARK-48702][INFRA] Fix `Python CodeGen check`

### What changes were proposed in this pull request?
The pr aims to fix `Python CodeGen check`.
https://github.com/panbingkun/spark/actions/runs/9647082015/job/26605246479#logs
<img width="986" alt="image" src="https://github.com/apache/spark/assets/15246973/0889e30a-4c5c-4d16-a996-8c237049cdf2">

### Why are the changes needed?
Make `Python CodeGen check` happy.



### Does this PR introduce _any_ user-facing change?
No.


### How was this patch tested?
- Pass GA.
- Manually test
```
python3.9 ./dev/connect-check-protos.py
...
+ cp gen/proto/python/spark/connect/types_pb2.py /var/folders/mb/yhrmjn4d44n5vp67_gbj8rh00000gn/T/check_connect_protos6tr54fq4
+ rm -Rf gen
Finish checking the generated codes in pyspark-connect: SUCCESS
```


### Was this patch authored or co-authored using generative AI tooling?
No.
